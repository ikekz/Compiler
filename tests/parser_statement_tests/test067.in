var
	i : double;
begin
	for i := 2 to 2 do
		begin
		end;
end.