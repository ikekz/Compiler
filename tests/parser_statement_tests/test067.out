(4,13) Error: Ordinal expression expected